packages/mobile/src/backup/utils.ts

```diff
@@ -1,37 +1,40 @@
-import { sampleSize } from 'lodash'
-import { AsyncStorage } from 'react-native'
+import * as _ from 'lodash'
 import { generateMnemonic, wordlists } from 'react-native-bip39'
-import { getKey, setKey } from 'src/utils/keyStore'
+import { getKey } from 'src/utils/keyStore'
 import Logger from 'src/utils/Logger'
 
 const TAG = 'Backup/utils'
 
+export const DAYS_TO_BACKUP = 1
+export const DAYS_TO_DELAY = 1 / 24 // 1 hour delay
+export const MNEMONIC_SPLITTER = 'celo'
```

Comment:
Do we want to use a word would blend in more with the rest of the key (while still being constant across languages)? Not sure if a user would think this was labeling which app the phrase belonged with vs part of the phrase

Reply:
I'm open to it. Any suggestions? The tricky part is finding a word that we're confident isn't in the bip39 word list for any language.
cc @nityas

Reply:
interesting-- are we sure celo isn't on this list for any language either(I think it has a translation both in spanish and slovenian)? We could use a separator like xxxx if not?